08.ls_object/lib/short_formatter.rb

```diff
@@ -8,36 +8,36 @@ def initialize(file_paths)
     @file_paths = file_paths
   end
 
-  def run
+  def format
     max_file_path_count = @file_paths.map { |f| File.basename(f).size }.max
     row_count = (@file_paths.count.to_f / COLUMN_COUNT).ceil
-    arranged_file_paths = align_array_column_of_file_paths
-    transposed_files = transpose(arranged_file_paths, row_count)
-    format_table(transposed_files, max_file_path_count)
+    file_paths = add_blanks_to_file_paths(@file_paths)
+    tableized_file_paths = tableize_file_paths(file_paths, row_count)
+    format_table(tableized_file_paths, max_file_path_count)
   end
 
   private
 
-  def align_array_column_of_file_paths
-    gap = @file_paths.count % COLUMN_COUNT
+  def add_blanks_to_file_paths(file_paths)
+    gap = file_paths.count % COLUMN_COUNT
     blank = gap.zero? ? 0 : COLUMN_COUNT - gap
     blanks = Array.new(blank, '')
     @file_paths + blanks
```

Comment:
```suggestion
    file_paths + blanks
```

のようにしてメソッド内ではインスタンス変数を参照しないようにした方がわかりやすいですね。